src/backend/src/services/change-requests.services.ts

```diff
@@ -579,12 +579,20 @@ export default class ChangeRequestsService {
     const project = createdCR.wbsElement.workPackage?.project || createdCR.wbsElement.project;
     const teams = project?.teams;
     if (teams && teams.length > 0) {
-      teams.forEach(async (team) => {
+      const completion: Promise<void>[] = teams.map(async (team) => {
         const slackMsg =
           `${type} CR submitted by ${submitter.firstName} ${submitter.lastName} ` +
           `for the ${project.wbsElement.name} project`;
-        await sendSlackChangeRequestNotification(team, slackMsg, createdCR.crId);
+        try {
```

Comment:
this try/catch should just be inside the sendSlackChangeRequestNotification method